pkg/status/status.go

```diff
@@ -5,13 +5,17 @@ const (
 	// merge configmaps.
 	ConfigmapMergeFailedStatus = "configmap-merge-failed"
 
-	// ResourceNotFoundStatus is set in the CR status when there is an failure during
-	// finding dependents kubernete resources.
-	ResourceNotFoundStatus = "resource-not-found"
+	// PendingStatus is set in the CR status when the kubeconfig secret
+	// or cluster values configmap does not exist yet.
+	PendingStatus = "pending"
 
 	// SecretMergeFailedStatus is set in the CR status when there is an failure during
 	// merge secrets.
 	SecretMergeFailedStatus = "secret-merge-failed"
+
+	// ValidationFailedStatus is set in the CR status when there is a validation error
+	// not related to dependent kubernetes resources.
+	ValidationFailedStatus = "validation-failed"
```

Comment:
Previously this was `resource-not-found` I changed it to `validation-failed` as that is more accurate.